odoo/http.py

```diff
@@ -159,21 +159,11 @@ def dispatch_rpc(service_name, method, params):
         odoo.tools.debugger.post_mortem(odoo.tools.config, sys.exc_info())
         raise
 
-def local_redirect(path, query=None, keep_hash=False, code=303):
-    # FIXME: drop the `keep_hash` param, now useless
+def local_redirect(path, query=None, code=303, open_redirect=False):
```

Comment:
let's rename+move it to `request.redirect_query()`, as the differentiator with `request.redirect()` is the extra `query` parameter being exposed, not the fact that local/relative redirections are possible

```py
    # In WebRequest
    def redirect_query(self, path, query=None, code=303, local=True):
        url = path
        if not query:
            query = {}
        if query:
            url += '?' + urls.url_encode(query)
        return self.redirect(url, code, local=local)
```